docs/how_to_guides/how_to_use_apparent_and_true_chemical_species.rst

```diff
@@ -0,0 +1,213 @@
+How to use apparent and true chemical species
+=============================================
+
+In ProteusLib, most all chemical processes simulated will be considered "true"
+species, i.e., species that actually exist in an aqueous solution. However, their
+may be times when you want to have your system report the "apparent" species
+between your inlet and/or outlet ports of a flowsheet. To do this, you need
+to understand how to define a species in a given configuration file as "apparent",
+which is a special component type in the **thermo-properties** configuration dictionary.
+
+For more information on creating a **thermo-properties** configuration dictionary,
+see 'How to setup simple chemistry'.
+
+
+What you need to update in the thermo-properties configuration dictionary
+-------------------------------------------------------------------------
+
+1. Add the ``"Apparent"`` and ``"StateIndex"`` objects under your import statements
+2. Specify specific species as ``"Apparent"`` within its component definition
+3. Give a list of "true" species it will dissociate into
+4. Define a "state_components" argument in the config and specify ``"StateIndex.true"``
+
+
+Example Configuration Dictionary
+--------------------------------
+
+This is best understood by going through an example. In this case, we will consider
+building off of the 'How to setup simple chemistry' guide and add ``"NaCl"`` as
```

Comment:
could add a hyperlink to this how-to (even though it's the doc right before this one in the toc tree)

Reply:
I wanted to do that, but trouble figuring it out. Is there a resource/guide for doing this?

Reply:
I also had trouble figuring this out. It ends up that doing the following would work:
```suggestion
building off of the :ref:`How to setup simple chemistry<how_to_setup_simple_chemistry>` guide and add ``"NaCl"`` as
```
So the syntax is :ref: `Label title<file_name>` in this case. You can also find more helpful info on Sphinx links [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks) and [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html).

Reply:
@adam-a-a It looks like after I made this change the building of the docs failed.

Reply:
Strange- I tested this locally before suggesting and didn't get any error; maybe I forgot to run `make clean` to start fresh before running `make html` again and missed the warning. Now when I try it locally, I also get the same error.

In any case, adding the following line to the top of the "how to setup simple chemistry" doc should (hopefully) resolve the issue:
`.. _how_to_setup_simple_chemistry:
`